examples/optimization/demo_MD.ipynb

```diff
@@ -0,0 +1,193 @@
+{
+ "cells": [
+  {
```

Comment:
I think this file leaked into the PR, to be removed.

Reply:
thanks